pkg/helper/client.go

```diff
@@ -159,12 +167,26 @@ func UpdateCurrentState(client client.Client, nodeNetworkState *nmstatev1alpha1.
 	return nil
 }
 
+func ping(target string) error {
```

Comment:
It may be interesting to log the output of the ping too. Don't you think?

Reply:
I want to see CI passing and then will cover this.